src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPlE A1419 iMac 27inch Retina 5k"
+      >
+
+      <h3 class="card__product-name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+      <p class="card__product-code">
+        Код товара: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+        </div>
+        <div>
+          Отзывов: 5
+        </div>
+      </div>
+
+      <div class="card__price">
+        <p>Цена:</p>
+        <p class="price-value">69 999 грн</p>
```

Comment:
У тебя нет блока price, поэтому это не может быть его элементом 
```suggestion
        <p class="card__value">69 999 грн</p>
```